### DISCUSSION TITLE
Anyone use KeePassXC with Sandboxie?

### DISCUSSION THREAD

**Discussion by e-t-l** [keepassxreboot/keepassxc#10154] [Q&A] [answered]
I use [Sandboxie-Plus](https://github.com/sandboxie-plus/Sandboxie/) for enhanced security between applications on Windows. I would like to have KeePassXC running in its own sandbox, but I would also like Firefox (also in its own sandbox) to be able to communicate it via the FF extension. Does anyone have this sort of setup working? What do your Sandboxie config files look like?

(Specifically, I can't get the Firefox extension to connect to the KeePassXC database unless they're launched in the same box. I don't know what files, protocols, or other components I would need to whitelist in order for the boxes to communicate with each other.)

**Comment by droidmonkey** [comment, marked as answer]
You need to do the following (untested just guessing):

**For Firefox:**
- Allow opening and running the keepassxc-proxy located at C:\Program Files\KeePassXC\keepassxc-proxy.exe
- Allow communication with named pipe `\Device\NamedPipe\org.keepassxc.KeePassXC.BrowserServer_%USERNAME%` (see https://sandboxie-plus.com/sandboxie/openpipepath/)

**For KeePassXC:**
- Allow opening and communication with the same named pipe as above

**Reply by e-t-l** [reply]
Wow, good guess! That worked perfectly.
